### PR TITLE
Update README docs

### PR DESCRIPTION
**Pull request by LuKks**
Closes https://github.com/holepunchto/hyperbee/issues/73

### CODE REVIEW

**Comment by LuKks** [comment]
Kind of unrelated but I think this should not indicate the `0` part:
https://github.com/holepunchto/hyperbee/blob/de6c3e6454bfa42b8361f1e25f520649e673bc1c/index.js#L468

So message stays the same as:
https://github.com/holepunchto/hyperbee/blob/de6c3e6454bfa42b8361f1e25f520649e673bc1c/index.js#L541

Proper error codes is also an option

File: README.md [hunk 1]
```diff
@@ -2,159 +2,185 @@
...
 
-#### `{ seq, key, value } = await db.get(key)`
+#### `const { seq, key, value } = await db.get(key)`
 
-Get a key, value. If the key does not exist, `null` is returned.
-`seq` is the hypercore version at which this key was inserted.
+Get a key's value. Returns `null` if key doesn't exists.
 
-#### `await db.del(key, [options])`
+`seq` is the `db.version` (Hypercore length) at which this key was inserted.
```
Comment (LuKks) [inline README.md:125]: I think it should be correct to refer `seq` as the `db.version` there

Comment (mafintosh) [reply README.md:125]: Just say hypercore index, no need to invent to many new concepts

**Comment by LuKks** [comment]
Reason on why updating the whole docs is to make it simpler, less big explanations, and more organized in general as there were many PRs from different people then overtime docs got a bit messy

**Pull request merged**